Content.Server/GameTicking/Rules/VariationPass/PoweredLightVariationPassSystem.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Content.Server.GameTicking.Rules.VariationPass.Components;
+using Content.Server.Light.Components;
+using Content.Server.Light.EntitySystems;
+using Content.Shared.Light.Components;
+using Robust.Shared.Random;
+
+namespace Content.Server.GameTicking.Rules.VariationPass;
+
+/// <summary>
+/// This handle randomly destroying lights, causing them to flicker endlessly, or replacing their tube/bulb with different variants.
```

Comment:
Isn’t it better to make a “replacement pass” that is not tied only to the light bulbs?  so that through it it would be possible just “no reinforced walls today” rule

Reply:
light code is giga sus so this is not just delete + spawn new at coords

Reply:
its probably possible to do to some extent